### DISCUSSION TITLE
Uncaught (in promise) SyntaxError: Unexpected token '<'

### DISCUSSION THREAD

**Discussion by faridsa** [alpinejs/alpine#535] [4. General]
Trying to use Alpine for the first time (v2.3.5) I goth that error in console under brave, safari, firefox developer and opera.
![Screen Shot 2020-05-28 at 12 48 07](https://user-images.githubusercontent.com/23212965/83163533-85237280-a0e1-11ea-82b4-ea6225d073bd.jpg)

**Comment by SimoTod** [comment]
It's likely a sintax error in one of your x-* attributes but it's hard to tell without seeing the code.

**Reply by faridsa** [reply]
Thank you so much for your wise advise. It really was what you said: an x-data sentence without closing quotes. No I know that this browser message is not related with Alpine but my own code.

**Reply by SimoTod** [reply]
No worries. Those errors are quite cryptic to be fair. 👍
